packages/node/src/common/StrictModeDetector.ts

```diff
@@ -0,0 +1,5 @@
+export class StrictModeDetector {
+    public static enabled() {
+        return !this;
+    }
+}
```

Comment:
I don't quite understand the purpose of this class. Would you mind explaining?

Reply:
Hey. Previously, because of my misunderstanding, I thought we need to verify if the node process is running in the strict env. However, this was a bad approach and I started checking node options passed by the user. The class is no longer here.